packages/react-reconciler/src/__tests__/ReactIncrementalErrorLogging-test.js

```diff
@@ -0,0 +1,231 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails react-core
+ * @jest-environment node
+ */
+
+'use strict';
+
+let React;
+let ReactNoop;
+
+describe('ReactIncrementalErrorLogging', () => {
+  beforeEach(() => {
+    jest.resetModules();
+    React = require('react');
+    ReactNoop = require('react-noop-renderer');
+  });
+
+  it('should log errors that occur during the begin phase', () => {
+    // Manually override console.error() because our toWarn() matchers
+    // filter out precisely the messages we want to test for in this file.
+    const oldConsoleError = console.error;
+    console.error = jest.fn();
```

Comment:
Why not do this override-and-restore stuff in `beforeEach`/`afterEach` ? Seems more "standard"

Reply:
Gotcha. I also don't feel strongly about this, was just curious.